serverless/messaging/how-to/monitor-mnq-cockpit.mdx

```diff
@@ -0,0 +1,57 @@
+---
+meta:
+  title: How to monitor Messaging and Queuing with Scaleway Cockpit
+  description: This page explains how to monitor Messaging and Queuing with Scaleway Cockpit
+content:
+  h1: How to monitor Messaging and Queuing with Scaleway Cockpit
+  paragraph: This page explains how to monitor Messaging and Queuing with Scaleway Cockpit
+tags: messaging queuing queueing cockpit metrics observability 
+categories:
+  - serverless 
+dates:
+  validation: 2023-09-07
+  posted: 2023-09-07
+---
+
+You can view your Messaging and Queuing metrics via [Scaleway Cockpit](/observability/cockpit/quickstart/). This allows you to monitor your queues/streams and messages at a glance. There are three steps to complete to view your Messaging and Queuing metrics for the first time with Cockpit:
+
+1. Activate your Cockpit
+2. Generate your Grafana credentials
+3. Access the Messaging and Queuing dashboard within Cockpit
```

Comment:
Isn't the dashboard in Grafana?

Reply:
I thought it's sort of Cockpit providing Grafana, but I can change to avoid confusion